assignment-1/pkg/controller/common.go

```diff
@@ -0,0 +1,17 @@
+package controller
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func GetSnapshotClassname() string {
+	return "csi-hostpath-snapclass"
+}
+
+func GetSnapshotName(pvc *corev1.PersistentVolumeClaim) string {
+	return pvc.Name + "-snapshot"
+}
+
+func GetApiGroup() string {
```

Comment:
```suggestion
func GetAPIGroup() string {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms